app/src/main/java/pl/droidevs/books/dao/BookDao.java

```diff
@@ -20,6 +22,9 @@
     @Query("SELECT * FROM " + BookEntity.TABLE_NAME)
     LiveData<List<BookEntity>> getAllBooks();
 
+    @Query("SELECT * FROM " + BookEntity.TABLE_NAME + " WHERE id = :bookId LIMIT 1")
```

Comment:
Uhu dlaczego zwracasz listę? Tylko jedna książka może mieć dane id. :)

Reply:
Nie udało mi się zrobić polecenia, które zwraca element. Tylko lista mi przechodzi, reszta - null :/

Reply:
hmm... w edycji mi działa, ale skoro tak to zmerguje i poprawię :)